style.css

```diff
@@ -734,6 +737,115 @@ textarea {
   .socials-end {
     padding-bottom: 2rem;
   }
+
+  /* Detail view */
+
+  .detail {
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    height: calc(100vh - 80px);
+    width: 70%;
+    overflow-x: hidden;
+  }
```

Comment:
Kindly update your popup window on your mobile version to match the figma design better🙏

Your design:
<img width="373" alt="image" src="https://github.com/axcell-0/microverse-portfolio/assets/37116322/c76ce8ba-ed92-4656-9f53-5f7017f550f4">

Figma design:
<img width="178" alt="image" src="https://github.com/axcell-0/microverse-portfolio/assets/37116322/5facbbe7-4177-4466-a725-5a53e6992891">

A few things to note:
- A lot of the elements in your mobile popup are overflowing horizontally. Please make sure to use responsive widths to ensure your elements are clearly visible as seen in the figma design
- The grey background behind the modal has some images in your design which are not present in the figma design. Please update your styles so they don't show
- Please make sure that ht modal buttons are properly aligned at the bottom of the modal instead of at the top and ensure that the contents of the button are center aligned.